src/TitleDetails/TitleDetailsContainer.js

```diff
@@ -89,6 +175,22 @@ TitleDetailsContainer.manifest = Object.freeze({
     accumulate: true,
     fetch: false,
   },
+  orderPieces: {
+    ...orderPiecesResource,
+    accumulate: true,
```

Comment:
IMHO, if you use resources you've just created only here, it makes sense to put `accumulate` and `fetch` inside a common code - I would expect we'll use stripes-connect that way in other places too